coursedata/texts/el.yaml

```diff
@@ -28,7 +28,7 @@ ui:
     commands_title: Εντολές
     by: "by"
 ClientErrorMessages:
-    Transpile_success: "Great job!"
+    Transpile_success: 'Great job'
```

Comment:
This entire line may be removed, and Hedy will then by default pick the English text